Moodlet/app/src/main/java/com/cmput3owo1/moodlet/fragments/MapFragment.java

```diff
@@ -23,64 +23,97 @@
     private static final String TAG = "Moodlet";
```

Comment:
I think each file needs an overarching comment as well

Reply:
Each class i mean

Reply:
Added description to the MapFragment class